### ISSUE TITLE
[iOS] Player overlay not accessible from VoiceOver

### ISSUE THREAD

**Issue by samglt** [jellyfin/Swiftfin#1733]
The overlay buttons in the video player can't be accessed by voiceover once they've faded out. This makes the player mostly unusable at best and usually impossible to exit. The way the native player handles it is by having a fullscreen "button" that toggles the overlay.

As an aside, this would also be helpful for automated screenshots/XCUITests since they behave similar to voiceover

**Comment by zoli211** [comment]
I have a branch that fixes this along with the rest of what VoiceOver runs into in the player: labels on the controls, an adjustable progress bar, and the controls no longer fading while they are being read. Opening a PR shortly.
